src/data/faq.json

```diff
@@ -1278,7 +1278,7 @@
                                 "textblock": [
                                     "You can reach us by phone from within Germany and from abroad. The national numbers are free of charge, but are only available from within Germany. The international numbers are <b>not free of charge</b> - the cost of the international call is based on the contract with your provider.",
                                     "<b>Technical Hotline</b>",
-                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Saturday from 7 a.m. to 10 p.m. German local time (except on German national holidays):",
+                                    "The technical hotline will help you with technical questions about the Corona-Warn-App, for instance if you have problems with the exposure logging. It is available Monday to Friday from 9 a.m. to 17 p.m. German local time (except on German national holidays):",

```

Comment:
I think it makes sense to add a note that the hotline is only available in German.

Reply:
ok, good idea!